static/app/views/organizationIntegrations/docIntegrationDetailedView.tsx

```diff
@@ -63,6 +63,10 @@ class DocIntegrationDetailedView extends AbstractIntegrationDetailedView<
     return this.integration.features ?? [];
   }
 
+  get requiresAccess() {
+    return false;
```

Comment:
Document integrations don't require access to install as they are installed & configured outside of Sentry

Reply:
Adding here because this class extends `AbstractIntegrationDetailedView`: `class DocIntegrationDetailedView extends AbstractIntegrationDetailedView`